Userland/Utilities/acpi.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2022-2022, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <stdio.h>
+#include <unistd.h>
+
+// https://uefi.org/specs/ACPI/6.4/05_ACPI_Software_Programming_Model/ACPI_Software_Programming_Model.html#system-description-table-header
+struct [[gnu::packed]] SDTHeader {
+    char sig[4];
+    u32 length;
+    u8 revision;
+    u8 checksum;
+    char oem_id[6];
+    char oem_table_id[8];
+    u32 oem_revision;
+    u32 creator_id;
+    u32 creator_revision;
+};
+
+void pkgLength(Vector<u8> const& data, u32& next_in_block, u32& next_block);
+
+void pkgLength(Vector<u8> const& data, u32& next_in_block, u32& next_block)
+{
+    u32 pos = next_block;
+
+    outln("");
+    out("data: ");
+    if (pos > 3)
+        for (auto i = -3; i < 0; i++)
+            out("{:#02x} ", data.at(pos + i));
+    else
+        out("               ");
```

Comment:
Braces around multiline if's

Reply:
Thank you! for the review!
This has been fixed